packages/smooth_app/test/users/registration_login_test.dart

```diff
@@ -40,7 +40,7 @@ void main() {
         User(userId: name, password: 'somerandomnewpassword'),
         queryType: queryType,
       );
-      expect(loginResponse?.successful, isNull);
+      expect(loginResponse?.successful, isFalse);
```

Comment:
Interesting, I was also getting confused at this when I had my first look and thought of raising an issue for that, I think the login process changed a bit.